packages/core/src/VNodes/VNode.ts

```diff
@@ -60,13 +66,12 @@ export class VNode {
         return this.renderingEngines[to].render(this) as T;
     }
     /**
-     * Locate where to set the range, when it targets this VNode, at a certain
-     * offset. This allows us to handle special cases.
+     * Transform the given DOM location into its VDocument counterpart.
      *
-     * @param domNode
-     * @param offset
+     * @param domNode DOM node corresponding to this VNode
+     * @param offset The offset of the location in the given domNode
      */
-    locateRange(domNode: Node, offset: number): [VNode, RelativePosition] {
+    locate(domNode: Node, offset: number): [VNode, RelativePosition] {
```

Comment:
I think it *might* be possible that a VNode return another VNode than itself in this function. @Zinston might have ideas of such nodes ?

Reply:
My first thought goes to atomic nodes. If we do end up having atomic groups or atomic nodes with shadow children (quarks? 😊), they would typically return another node than themselves. Eg: a child of an iframe would return the iframe or one of its siblings.
I do think it would be bad practice though and to be avoided as much as possible. If we can ensure that our always returns `this`, it would be great, though in that case indeed just returning the relative position would suffice. I'm talking ideally here but I know we've had plenty of discussions on atomic nodes where we had disagreements on their definition and scope.

Reply:
What's the status here finally?

Reply:
I'm keeping `locate` for now. I like the 1<->1 conversion between `[Node, number]` and `[VNode, RelativePosition]`.